firebase-config/firebase-config.gradle

```diff
@@ -24,6 +24,8 @@ firebaseLibrary {
     publishSources = true
 }
 
+// Compile Protobuf definition files. Remote Config does not depend on Protobuf but includes
+// definitions used by the ABT and FIAM SDKs.
```

Comment:
afaict I don't think that is accurate, instead ABT has protos used by fiam and RC, can you please double check that?

Reply:
ABT only exposes `experiment_payload` proto, which is only used in RC in the class removed in this PR. There are other mentions in the configs proto, but it's as bytes

https://osscs.corp.google.com/firebase-sdk/firebase-android-sdk/+/master:firebase-config/src/proto/com/google/android/gms/config/proto/config_persistence.proto;l=42?q=experiment%20payload&ss=firebase-sdk%2Ffirebase-android-sdk:firebase-config%2F

FIAM does depend on the class
https://osscs.corp.google.com/firebase-sdk/firebase-android-sdk/+/master:firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/internal/AbtIntegrationHelper.java;l=57?q=experimentpayload&ss=firebase-sdk%2Ffirebase-android-sdk:firebase-inappmessaging%2F from ABT